packages/ui/client/composables/client/static.ts

```diff
@@ -4,7 +4,7 @@ import type { WebSocketHandlers } from 'vitest/src/api/types'
 import { parse } from 'flatted'
 import { decompressSync, strFromU8 } from 'fflate'
 import type { File, ModuleGraphData, ResolvedConfig } from 'vitest/src/types'
-import { StateManager } from '../../../../vitest/src/node/state'
```

Comment:
How is this related to documentation?

Reply:
Mistake. i'll revert this and open a new PR for it.